model_analysis/README.md

```diff
@@ -1,15 +1,18 @@
 # Model Analysis
-Here provide jupyter notebooks, which contains performance visualization such as GradCAM images (for checkpoint models), superclass performance, model cascades and oracle upper bound.  
+We provide Jupyter notebooks, which contain performance visualization via GradCAM images (for checkpoint models), superclass performance, model cascades and oracle upper bound.  
 
 #### [GradCAM.ipynb](GradCAM.ipynb)
+This notebook visualizes model attribution for each image. As required preprocessing, we store each image as a torch tensor, arranged class-wise. Note that the example in the notebook is not the one shown in the paper.
 
-This notebook visualizes model attribution for each image. We beforehand store each image as torch tensor, arranged class-wise, and therefore please do so before running this script. Note that the example in the notebook is not the one shown in the paper.
+**TODO @ GB** please add a comment why and how it is different from paper
```

Comment:
I've taken care of this in the main branch. There will be a slight merge conflict since I changed the GradCAM.ipynb and README.md.